Sources/MapboxNavigation/CarPlaySearchController.swift

```diff
@@ -1,6 +1,105 @@
 import CarPlay
+import MapboxGeocoder
 import MapboxDirections
 
+public struct NavGeocodedPlacemark: Equatable, Codable {
```

Comment:
This struct will also have to be documented, probably also makes sense to rename it to `NavigationGeocodedPlacemark`.